resources/webviews/aksclusterproperties/clusterproperties.html

```diff
@@ -7,6 +7,34 @@
     <title>AKS Cluster Properties</title>
     <!-- Link to the css file -->
     <link rel="stylesheet" href="{{cssuri}}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://use.fontawesome.com/7dddd54b5c.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+
+    <script>
+        // https://code.visualstudio.com/api/extension-guides/webview#passing-messages-from-a-webview-to-an-extension
+        const vscode = acquireVsCodeApi();
+
+        $(document).ready(function () {
+            $('#Startcluster').click(function () {
+                $(".spinner").show();
+                $(this).hide();
+                vscode.postMessage({ command: "startCluster" });
+                // https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-blob-service-operations
+                setTimeout(function() {$(".spinner").hide()}, 30000);
```

Comment:
Do we need to hide the spinner at all?

First off, I thought the extension could notify the webview when the operation finished, and then it could hide the spinner.

But then I noticed that when the operation finishes, the entire webview html gets replaced. So effectively the spinner will be shown for the rest of the current document's life, and I *think* that means we don't need to hide it?